src/deep-security/lib/IdentityProvider.js

```diff
@@ -13,46 +13,39 @@ import {InvalidProviderIdentityException} from './Exception/InvalidProviderIdent
  */
 export class IdentityProvider {
   /**
-   * @returns {string}
+   * @param providerName
+   * @param providers
+   * @returns {*}
    */
-  static get AMAZON() {
-    return 'www.amazon.com';
-  }
-
-  /**
-   * @returns {string}
-   */
-  static get FACEBOOK() {
-    return 'graph.facebook.com';
-  }
-
-  /**
-   * @returns {string}
-   */
-  static get GOOGLE() {
-    return 'accounts.google.com';
-  }
-
-  /**
-   * @param {String} providerName
-   * @returns {Array}
-   */
-  static ALIASES(providerName) {
-    let aliases = [];
+  getProviderDomain(providerName, providers) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/MitocGroup/deep-framework/file/2349020820/issues/source?bid=2895475&fileBranchId=3217145#l20)